Simplenote.xcodeproj/project.pbxproj

```diff
@@ -1590,26 +1590,11 @@
 			files = (
 			);
 			inputFileListPaths = (
-			);
```

Comment:
They are very useful and a welcome addition from Xcode 11!

_FYI, SwiftGen has migrated to them now too, so that you can make the `swiftgen` Script Build Phase only run if one of the file listed in the xcfilelist` has changed, and SwiftGen itself is able to generate that `xcfilelist` text file from its config file and thus from the input files the codegen ends up depending on. All that without SwiftGen having to modify the `pbxproj`, just an external `xcfilelist`… very useful!_

Reply:
Neat